graphql-kotlin-schema-generator/src/main/kotlin/com/expediagroup/graphql/generator/types/generateMutation.kt

```diff
@@ -46,6 +47,9 @@ internal fun generateMutations(generator: SchemaGenerator, mutations: List<TopLe
             .forEach {
                 val function = generateFunction(generator, it, generator.config.topLevelNames.mutation, mutation.obj)
                 val functionFromHook = generator.config.hooks.didGenerateMutationField(mutation.kClass, it, function)
+                if (mutationBuilder.hasField(functionFromHook.name)) {
```

Comment:
wondering whether we should keep mapping of fields to file names - it would provide much more meaningful error message but wondering whether this extra complexity is needed

@smyrick any thoughts?

Reply:
At first, I also considered keeping mapping information, but it is too much unnecessary code only for error message,

or We could just build the type builder and get the definition, but I don't think this is appropriate neither.

Reply:
I think this is straight forward enough to just fail on first conflict